src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -767,14 +767,14 @@ def load_arguments(self, _):
             c.argument('license_type', help=license_msg, arg_type=get_enum_type(['Windows_Server', 'Windows_Client', 'None']))
             c.argument('priority', resource_type=ResourceType.MGMT_COMPUTE, min_api='2019-03-01',
                        arg_type=get_enum_type(self.get_models('VirtualMachinePriorityTypes'), default=None),
-                       help="Priority. Use 'Spot' to run short-lived workloads in a cost-effective way. 'Low' enum will be deprecated in the future. Please use 'Spot' to deploy Azure spot VM and/or VMSS")
+                       help="Priority. Use 'Spot' to run short-lived workloads in a cost-effective way. 'Low' enum will be deprecated in the future. Please use 'Spot' to deploy Azure spot VM and/or VMSS. Default to Regular.")
```

Comment:
can we add link to service doc of deprecation detail?

Reply:
> 
> 
> can we add link to service doc of deprecation detail?

Prerequisite is it has a doc showing deprecation detail. It depends on when swagger changes.
arg_type=get_enum_type(self.get_models('VirtualMachinePriorityTypes') can fetch enum values.